### ISSUE TITLE
Terminal colors don't reset after td-list

### ISSUE THREAD

**Issue by perlsaiyan** [darrikonn/td-cli#8]
Terminal colors are not reset correctly after running "td list" or plain "td", at least in the case where there are no todo items.

**Comment by darrikonn** [comment]
Thanks for submitting the issue!
I'll take a look!

**Comment by darrikonn** [comment]
Can you tell me what version you're running?

```bash
td --version
```

**Comment by darrikonn** [comment]
I believe that you're running an older version. This used to be a problem when firing up interactive mode with no todo items.
I made a new release that fixed this problem and added new interactive commands as well.

Can you verify that for me?
`td --version` should yield version `1.1.0`
```
[in]  td --version
[out] td version 1.1.0 - (C) Darri Steinn Konn Konradsson
```

If you're not running that version, you can update with
```
pip3 install td-cli --upgrade
```

**Comment by darrikonn** [comment]
I'm going to close this for now (since I believe version mismatch is the issue). Feel free to open this again if upgrading does not solve your issue 👍

**Comment by perlsaiyan** [comment]
Sorry about that, I am running 

td --version
td version 1.1.0 - (C) Darri Steinn Konn Konradsson

I'm using alacritty, i also tested xterm.

**Comment by darrikonn** [comment]
Thanks for that. I'll investigate, since this is not a version issue.
I'll keep you updated!

**Comment by darrikonn** [comment]
Could you submit gif/screenshots when this occurs? Preferably running with the verbose flag `td --verbose`.
I can't seem to reproduce this 🤔. I ran this on `alacritty` and `iTerm` with fresh install and clean `td-cli` database.

**Comment by darrikonn** [comment]
@perlsaiyan I was finally able to reproduce this on `termux` on Android. Found out the issue and created a new patch version with the fix (1.2.2). If you upgrade td-cli, it should be fixed: `pip3 install --upgrade td-cli`.

Thanks!
